pkgs/development/python-modules/opencamlib/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  scikit-build-core,
+  cmake,
+  ninja,
+  stdenv,
+  llvmPackages,
+  boost,
+  python,
+}:
+
+buildPythonPackage rec {
+  pname = "opencamlib";
+  version = "2023.01.11";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aewallin";
+    repo = "opencamlib";
+    tag = version;
+    hash = "sha256-pUj71PdWo902dqF9O6SLnpvFooFU2OfLBv8hAVsH/iA=";
+  };
+
+  build-system = [
+    scikit-build-core
+    cmake
+    ninja
+  ];
+
+  buildInputs = lib.optionals stdenv.cc.isClang [ llvmPackages.openmp ];
+
+  dependencies = [ boost ];
```

Comment:
`boost` should be in `buildInputs`, `dependencies` is specifically for python package dependencies.